ob-pattern-1/templates/obkm/obkm-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2-obkm-deployment
+  namespace: wso2
+spec:
+  replicas: 1
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      deployment: wso2-obkm
+  template:
+    metadata:
+      labels:
+        deployment: wso2-obkm
+    spec:
+      containers:
+      - name: wso2-obkm
+        image: docker.wso2.com/wso2-obkm:1.4.0
+        livenessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - nc -z localhost 9446
+          initialDelaySeconds: 250
+          periodSeconds: 10
+        readinessProbe:
+          exec:
+            command:
+              - /bin/sh
+              - -c
+              - nc -z localhost 9446
+          initialDelaySeconds: 250
+          periodSeconds: 10
+        lifecycle:
+          preStop:
+            exec:
+              command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/wso2server.sh stop']
+        imagePullPolicy: Always
+        securityContext:
+          runAsUser: 802
+        ports:
+        - containerPort: 9763
+          protocol: TCP
+        - containerPort: 9446
+          protocol: TCP
+      serviceAccountName: "wso2svc-account"
+      imagePullSecrets:
+      - name: wso2ob-deployment-creds
+     
+
```

Comment:
@kdanW remove this additional EOF line. Only one is required.

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/2b2f6f4c2e0c8feada3a3eda00548a132db73a26